packages/flutter/test/cupertino/switch_test.dart

```diff
@@ -526,6 +526,43 @@ void main() {
     expect(find.byType(CupertinoSwitch), paints..rrect(color: trackColor));
   });
 
+  testWidgets('Switch is using default thumb color', (WidgetTester
+  tester) async {
+    await tester.pumpWidget(
+      const Directionality(
+        textDirection: TextDirection.ltr,
+        child: Center(
+          child: CupertinoSwitch(
+            value: false,
+            onChanged: null,
+          ),
+        ),
+      ),
+    );
+
+    expect(find.byType(CupertinoSwitch), findsOneWidget);
+    expect(tester.widget<CupertinoSwitch>(find.byType(CupertinoSwitch)).thumbColor, CupertinoColors.white);
```

Comment:
Could you also check the thumb is painted in the right color, similar to line 526?

Reply:
I planned to do this, but I don’t know how to test the thumb painter. Is there an example that I can refer to? thank you!

Reply:
Thanks for reminding, I will add thumb paint test.

Reply:
done!